src/MCRun.jl

```diff
@@ -280,7 +567,23 @@ Step size adjustment is done after `n_adjust` MC cycles.
 Evaluation: including calculation of inner energy, heat capacity, energy histograms;
 saved in `results`.
 """
-function ptmc_run!(mc_states, move_strat, mc_params, pot, ensemble, results)
+function ptmc_run!(mc_states, move_strat, mc_params, pot, ensemble, results; save_ham::Bool = true, save::Bool=true, restart::Bool=false,restartindex::Int64=0,save_dir = pwd())
+    #restart isn't compatible with saving the hamiltonian at the moment
+
```

Comment:
restart not yet complete

Reply:
What's left to be done?

Reply:
RDF and histograms haven't been checked on restart

Reply:
ok - keep this on your to-do-list :)